docs/tutorial/C++/cplusplus_interface.rst

```diff
@@ -0,0 +1,51 @@
+.. _cplusplus_interface_tutorial:
+
+C++ interface
+-------------
+
+This page explains how to create a CMake based C++ project using the Open3D C++ interface.
+
+.. note:: For help on the C++ interfaces, refer to examples in [src/Test] and [src/Tools] folder and check `Doxygen document for C++ API <http://open3d.org/cppapi>`_.
+
+.. _install_open3d_from_source:
+
+Install open3d from source
+==========================
+
+For installing from source see :ref:`getting_started_compilation`.
+
+.. _create_cplusplus_project:
+
+Create C++ project
+==================
+
+.. warning:: The following is supported at this time only on Ubuntu with all dependencies installed.
+
+Let's create a basic C++ project based on CMake and Open3D installed libraries and headers.
+
+1. Get the code from :download:`TestVisualizer.cpp <../../_static/C++/TestVisualizer.cpp>`
+2. Get the CMake config file from :download:`CMakeLists.txt <../../_static/C++/CMakeLists.txt>`
+3. mkdir -p build
+4. cd build
+5. cmake ..
+6. make -j
```

Comment:
How about make it like ``make -j``? These are shell commands.

Reply:
got it.